### ISSUE TITLE
Touchscreen/wacom drivers for the X390 YOGA?

### ISSUE THREAD

**Issue by Pajamaland** [mendax1234/ThinkpadX390-Opencore-EFI#20]
Would you consider integrating the drivers for the X390 YOGA into the next OC release?

**Comment by mendax1234** [comment]
> Would you consider integrating the drivers for the X390 YOGA into the next OC release?

Hi, since l mine is not X390 Yoga, it will be hard for me to test whether the drivers will work normally for the X390 YOGA. I hope you can try it yourself and if you encounter any issues, feel free to ask here. Thanks!

**Comment by Pajamaland** [comment]
What drivers should I search for? Is there a good place to look?

**Comment by mendax1234** [comment]
> What drivers should I search for? Is there a good place to look?

Yeah, there are many useful forums you can use, for example, [Tonymacx86](https://www.tonymacx86.com/),  [Insanelymac](https://www.insanelymac.com/) and so on.
